src/test/scala/akka/wamp/ManagerSpec.scala

```diff
@@ -56,7 +58,46 @@ class ManagerSpec
     pending
   }
 
-  it should "Fail on wrong message format" in { f =>
+  it should "Receive text messages and emit Events" in { f =>
```

Comment:
I agree that the responsibility for recovering from a message with a wrong format belongs to the transport Actor.

The Manager is where the current code for creating this actor sits:
```scala
val serializer = Serializers.streams(subprotocol)
val serialiserStrategy = ActorAttributes.supervisionStrategy(messageFlowDecider)
val (outgoingActor, upgradeResponse) =
	outgoingSource
	  .via(serializer.serialize.withAttributes(serialiserStrategy))
	  .viaMat(webSocketFlow)(Keep.both)
	  .viaMat(serializer.deserialize.withAttributes(serialiserStrategy))(Keep.left)
	  .toMat(incomingSink)(Keep.left)
	  .run()
```
So I added the tests to the ManagerSpec as the code for utilising the passed Descider is there.
It is really  not easy to test the Manager as it Relies to a lot of objects including the Http and Serializers which are not easy to mock.

There should be a way to test all the code in the Manager which is what I was trying to achieve.

Perhaps the tests names are not correct or the Manager way to create the Serialiser stream should be configurable.

It looks quite convenient to be able to pass what Decider to be used in the Connect message:
```
case cmd @ Wamp.Connect(client, uri, subprotocol, messageFlowDecider) => {
```

This way we don't have to create a different instance of IO(Wamp) if we want to have different error handling strategies for the different connections.